testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -21,7 +21,7 @@ Feature: Enable and disable monitoring of the server
     And file "/etc/sysconfig/tomcat" should not contain "Dcom.sun.management.jmxremote.port=3333 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=" on server
     And file "/etc/rhn/taskomatic.conf" should not contain "Dcom.sun.management.jmxremote.port=3334 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=" on server
 
-  Scenario: Restart spacewalk services to apply monitoring config changes
+  Scenario: Restart spacewalk services to apply config changes after disabling monitoring
     When I restart the spacewalk service
 
   Scenario: Check that monitoring is disabled using the UI
```

Comment:
Off-topic:
```
    And I should see a list item with text "System" and bullet with style "fa-check text-success"
    And I should see a list item with text "PostgreSQL database" and bullet with style "fa-check text-success"
    And I should see a list item with text "Server self monitoring" and bullet with style "fa-hand-o-right text-success"
    And I should see a list item with text "Taskomatic (Java JMX)" and bullet with style "fa-hand-o-right text-success"
    And I should see a list item with text "Tomcat (Java JMX)" and bullet with style "fa-hand-o-right text-success"
``` 
It might be good to change, as we did in other steps, the with style "xxxxx"  by some more natural language

Reply:
Oh yes!!!! +10000 !!!!

Reply:
https://github.com/SUSE/spacewalk/issues/11966